src/main/java/duke/LoadFile.java

```diff
@@ -0,0 +1,57 @@
+package duke;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Scanner;
+import java.io.File;
+
+public class LoadFile {
+    public static boolean loaded = false;
```

Comment:
boolean variable should be name in a way that sound like boolean. For example 'boolean hasLoaded'